src/edu/stanford/nlp/sempre/Main.java

```diff
@@ -27,9 +28,16 @@ public void run() {
       Server server = new Server(master);
       server.run();
     }
+    
+    if (interactiveLearning) {
```

Comment:
Can we not have any code that specializes to `interactive`?  Ideally, we'd just instantiate the `InteractiveMaster` class via reflection which takes care of setting everything else up...this way we don't clutter `Main.java`.